Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptDeleteController.java

```diff
@@ -139,7 +143,7 @@ public ModelAndView confirmlDelete(@PathVariable("id") String id,
             model.setViewName("redirect:/login");
             return model;
         }
-        model.setViewName("/auth/conceptlist/concepts");
+        model.setViewName("forward:/auth/" + listName + "/concepts");
```

Comment:
I think 'redirect' is the better way to go here. Otherwise you're browser will still show the previous url and a reload of the page would call the delete method again.